src/UserModule.cc

```diff
@@ -37,7 +37,7 @@
 
 std::deque<std::string> UserModule::module_stack;
 
-static void __attribute__ ((noinline)) print_err(std::string name, const Location &loc,const Context *ctx){
+static void /*__attribute__ ((noinline))*/ print_err(std::string name, const Location &loc,const Context *ctx){
```

Comment:
for cross-reference: https://github.com/openscad/openscad/pull/2788#discussion_r262237444

Reply:
Changed to use `compiler_specific.h` / `NOINLINE` as suggested.